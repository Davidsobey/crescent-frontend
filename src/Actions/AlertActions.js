import AlertConstants from '../Constants/AlertConstants';

function success(message) {
  return { type: AlertConstants.SUCCESS, message };
}

function error(message) {
  return { type: AlertConstants.ERROR, message };
}

function clear() {
  return { type: AlertConstants.CLEAR };
}

const AlertActions = {
  success,
  error,
  clear,
};

export default AlertActions;
