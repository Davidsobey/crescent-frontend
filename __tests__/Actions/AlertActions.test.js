import configureStore from 'redux-mock-store';

// Actions to be tested
import AlertActions from '../../src/Actions/AlertActions';

const mockStore = configureStore();
const store = mockStore();

describe('AlertActions', () => {
  beforeEach(() => { // Runs before each test in the suite
    store.clearActions();
  });

  test('Dispatches the correct action and payload', () => {
    expect(store.getActions()).toMatchSnapshot();
  });

});