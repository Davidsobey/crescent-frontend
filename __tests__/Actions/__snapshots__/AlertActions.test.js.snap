// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertActions Dispatches the correct action and payload 1`] = `Array []`;
